docs/src/main/tut/symbols.md

```diff
@@ -0,0 +1,19 @@
+| Symbol | Name                   | Typeclass      | Import                       | Source                                              |
+| ------ | ---------------------- | ------------   | --------------------------   | -------------------------------------------------   |
+| `<*>`  | apply                  | `Apply`        | TBA                          | core/src/main/scala/cats/Apply.scala                |
+| `|@|`  | Cartesian builder      | `Cartesian`    | `cats.syntax.cartesian._`    | core/src/main/scala/cats/Cartesian.scala            |
```

Comment:
@non I agree with getting rid of the imports and having an example usage.

I'm undecided about what to do with the source file.  People often want to know where a symbol is defined, but because most of them use simulacrum or macro expansions, there isn't a straightforward location.  @DavidGregory084 wouldn't references to `src_managed` confuse people, as it isn't present on GitHub?  Perhaps a link to the source file or syntax file would be better.
I'm leaning towards:

```
|Symbol    | Name       | Typeclass                                             | Definition |
| -------   | --------  | ------------------------------------------------------ | ------------- |
|  x =!= y  |  not equals  | [cats.kernel.Eq](../../../../cats/kernel/Eq.scala)  | Eq[A].neqv(x, y) |
```